config/cmake/MFEMConfig.cmake.in

```diff
@@ -11,7 +11,10 @@
 
 include(${CMAKE_CURRENT_LIST_DIR}/MFEMConfigVersion.cmake)
 
+set(MFEM_MPIRUN_CMD @MFEM_MPIRUN_CMD@)
```

Comment:
Maybe I'm missing something but I don't see a reason to define any MPI related variables in `MFEMConfig.cmake`.

For the purposes of testing mfem with cmake/ctest, we already use the "standard" cmake variables `MPIEXEC`, `MPIEXEC_NUMPROC_FLAG`, etc. That way, a user can control the behavior using the "standard" MPI cmake variables.